src/linear_algebra/basic.lean

```diff
@@ -201,7 +202,8 @@ coe_injective.unique
 
 /-- The sum of two linear maps is linear. -/
 instance : has_add (M →ₗ[R] M₂) :=
-⟨λ f g, ⟨λ b, f b + g b, by simp [add_comm, add_left_comm], by simp [smul_add]⟩⟩
+⟨λ f g, { to_fun := f + g,
+          map_add' := by simp [add_comm, add_left_comm], map_smul' := by simp [smul_add] }⟩
```

Comment:
The change from `λ b, f b + g b` to `f + g` broke some downstream proofs, but if CI passes then I fixed all of those too.

Reply:
Can you give an example of a broken proof? Perhaps a neat solution is to add a `coe_add : (f + g : M → M₂) = f + g` lemma which you can rewrite with instead of unfolding `has_add.

Reply:
I've commented on two cases below.